styles.css

```diff
@@ -553,4 +1155,228 @@ textarea {
+    padding: 20px;
+    border-radius: 0;
+    margin-left: 0;
+  }
+
+  .form-group {
+    margin-bottom: 10px;
+  }
+
+  input[type="text"],
+  input[type="email"],
+  textarea {
+    width: 100%;
+    padding: 10px;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+    box-shadow: 0 0 3px rgba(0, 0, 0, 0.1);
+    font-size: 16px;
+    background: #fff;
+    box-sizing: border-box;
+  }
+
+  textarea {
+    height: 75px;
+  }
+
+  .input-text {
+    font-family: 'Inter', sans-serif;
+    font-style: italic;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    color: #6b778c;
+  }
+
+  .input-message {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    height: 75px;
+    color: #42526e;
+  }
+
+  .email {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    margin-top: 0;
+    color: #42526e;
+  }
+
+  .btn-2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    margin-top: 5%;
+    height: 40px;
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    background-image: url('img/Header-llustration-desktop.png');
+    background-repeat: no-repeat;
+    background-size: 100% auto;
+  }
+
+  .nav {
+    display: flex;
+    justify-content: center;
+  }
+
+  .nav li {
+    margin: 0 40px;
+  }
+
+  .nav ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+    color: #505f79;
+    line-height: 0.5;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+  }
+
+  #about-id {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    overflow: hidden;
+    margin-top: 10%;
+  }
+
+  .image-group {
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    align-content: flex-start;
+    content: "";
+    width: 287.84px;
+    height: 150px;
+    margin-left: -35%;
+    margin-top: -15%;
+    background-image: url('img/Group.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: left 0;
+    z-index: 1;
+  }
+
+  .image-group-1 {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    align-content: flex-end;
+    content: "";
+    width: 269.55px;
+    height: 244.78px;
+    margin-left: 110%;
+    margin-top: -25%;
+    background-image: url('img/Group1.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right 0;
+    z-index: 2;
+  }
```

Comment:
- [ ] As a result of using the div tags for the images it pushes the whole design to the left side thereby cutting part of it off hence it is not centered. I believe fixing the background images as recommended above will help fix this issue. Kindly check [this reference](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html) on when to use an img tag and when to use css background image to help achieve the best standard practices. :wink: 

![Screenshot from 2023-06-08 08-07-28](https://github.com/NgachaIan/Personal_Portfolio/assets/65233860/b94c1257-e007-4774-b0d2-72686987603b)

Reply:
@jerryowusu from the design, how can you tell when an image is a background image or not? Apart from the icons, I think all images were background images

Reply:
Yes the ones on top of the Languages, Frameworks, and Skills are suppose to use the img tags instead of the background images as I recommended on lines 250 to 254 (index.html file). However, the major issue here in this instance is creating empty divs and applying the background images to those divs as you have done in some parts of your index.html file. If you know it is a background image, kindly apply it as the background image of the respective container and not create empty divs for that. There are some instances where you created empty divs, gave it a width and height just to apply a background image (lines 141 - 142 in index.html file, and lines 951 - 1000 in style.css). This is not a good practice since it will maintain the stated width and height even if the div is empty.
I believe it will help you to grasp the concepts of using the img tags and the css background image if you take your time and go through [this reference](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html)

Reply:
@jerryowusu thank you I have made all changes, now there are no empty divs or empty divs with background images in css but the site is yet to be responsive in various desktop screen sizes. Am working on that now, been working on that since yesterday